docs/content/reports/ransomware.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Ransomware report"
+date: 2021-08-05T13:23:10+03:00
+draft: false
+description: "Provides information about ransomware simulation on your network"
```

Comment:
I'm thinking we should explicitly set the order of the report pages in the sidebar.
![image](https://user-images.githubusercontent.com/44770317/128539943-ac803f50-3603-4a73-b5f9-d866fe09c110.png)

Let's add `weight` here (and in the other report pages) to control the order. My suggestion would be: 
Security
Zero Trust
ATT&CK
Ransomware